package.json

```diff
@@ -4,18 +4,50 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
-    "dev": "vite",
+    "dev": "vite --open --host",
+    "preview": "vite preview",
     "build": "vite build",
-    "preview": "vite preview"
+    "test": "react-scripts test"
```

Comment:
- [ ] Kindly add more tests as suggested earlier.

- [ ] I am unable to run your existing tests. When I run `npm test` I get this error message. Kindly fix that.

  ![image](https://user-images.githubusercontent.com/54848363/212292183-d001e641-5d91-4827-a2ca-aa9c419bbc55.png)